### DISCUSSION TITLE
Flutter web

### DISCUSSION THREAD

**Discussion by jmurrayrs** [Flutterando/forum#167] [Discussões gerais]
Olá pessoal ! Existe alguma forma de gerar um site web que não seja "containerizado" dentro de um componente "flt-glass-pane" ? Fala sério, parece que estamos rodando um projeto em Flash !

**Comment by eduardoflorence** [comment]
Para isso você tem que usar a flag `--web-renderer`, especificando o valor `html` na hora de executar ou fazer o build do seu app.
Veja mais detalhes aqui: https://docs.flutter.dev/platform-integration/web/renderers

**Reply by jmurrayrs** [reply]
Olá Eduardo ! Agradeço pelo interesse em ajudar. Sim, eu conheço esse comando, mas o que está me deixando incomodado é esse ar de "Flash" que a aplicação web fica. Um container "flutter-view" com tudo lá dentro, renderizado pelo que, o canvas kit ? 

![image](https://github.com/Flutterando/forum/assets/4536254/6eed36a1-2890-44bf-8101-101d7ecbb43b)

**Reply by eduardoflorence** [reply]
@jmurrayrs você executou este comando no terminal:
```
flutter build web --web-renderer html
```
E o html gerado na pasta `./build/web/` ficou dessa forma que está na imagem?

**Reply by jmurrayrs** [reply]
Sim, ficou, até subi uma demo...

https://haidai.com.br/flutter/

O html de index.html ficou desse jeito

![image](https://github.com/Flutterando/forum/assets/4536254/98e6f682-7547-41a0-a17f-7f53d2d35153)

Não curti a proposta. Essa renderização ficou ruin. Como vou ranquear isso no google, por exemplo ?
